spec/support/fake_bisect_runner.rb

```diff
@@ -1,3 +1,5 @@
+require 'set'
```

Comment:
Do you mind using `RSpec::Core::Set` rather than stdlib `Set`?

We try hard to minimize how much of the stdlib we depend on (to limit situations where gem authors could have passing RSpec suites while forgetting to load a bit of stdlib they use) and defined `RSpec::Core::Set` specifically for situations where we want a set-like collection.